src/main/java/frc/robot/commands/DriveToNoteWithSearch.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.chassis.Chassis;
+
+public class DriveToNoteWithSearch extends Command {
+  private final Chassis chassis;
+
+  NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+  NetworkTableEntry tx = table.getEntry("tx");
+  NetworkTableEntry ty = table.getEntry("ty");
+  NetworkTableEntry tv = table.getEntry("tv"); // 1 if target is deteced, 0 if not.
+
+  private final PIDController translationController = new PIDController(0.07, 0, 0);
+  private final PIDController rotationController = new PIDController(0.07, 0, 0);
+  private final double searchRotationSpeed = 0.07;
```

Comment:
The units for this speed are radians/second

We probably want to go 90° in ~ 2 seconds, so that would be pi/2 radians/2 seconds, or pi/4 radians / second